client/components/telecom/telephony/abbreviatedNumbers/telecom-telephony-abbreviated-numbers.modal.html

```diff
@@ -1,171 +1,77 @@
 <form name="abbreviatedNumberForm"
       id="abbreviatedNumberForm"
-      data-ng-submit="AbbreviatedNumberModal.send()"
-      novalidate>
+      novalidate
+      data-ng-submit="abbreviatedNumberForm.$valid && AbbreviatedNumberModal.send()">
```

Comment:
does it mean that users will be abble to click on the validate button even if the form is invalid ?

Reply:
yes as specified by guidelines